app/views/layouts/_getready_incompany.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="container-fluid">
+  <div class="jumbotron" align="middle">
+    <h2> Get Ready for the Day </h2>
+    <h3 class="lead"> Requirements</h3>
+    <p class="paragraph">Here goes all kinds of awesome text that will describe what is required of attendees to attend this amaaaazing event!</p>
+  </div>
+</div>
```

Comment:
Also you should include the file in which you are rendering this. I guess your incompany page...